src/v2/repositories/implementations/xcm/ClvXcmRepository.ts

```diff
@@ -0,0 +1,244 @@
+import { BN } from '@polkadot/util';
+import { decodeAddress } from '@polkadot/util-crypto';
+import { XcmTokenInformation } from 'src/modules/xcm';
+import { container } from 'src/v2/common';
+import { ExtrinsicPayload, IApi, IApiFactory } from 'src/v2/integration';
+import { Asset } from 'src/v2/models';
+import { Symbols } from 'src/v2/symbols';
+import { XcmRepository } from '../XcmRepository';
+import { XcmChain } from 'src/v2/models/XcmModels';
+import { Option, Struct } from '@polkadot/types';
+import { getPubkeyFromSS58Addr } from 'src/hooks/helper/addressUtils';
+
+interface TokensAccounts extends Struct {
```

Comment:
I believe this struct is already defined somewhere else. If so, please reuse

Reply:
The TokenAccounts struct looks different on different chains(Acala/Crust). Currently it looks similar to CrustXcmRepository. But it's a little bit wired to introduce the CrustXcmRepo dependency for CLVXcmRepo...

Reply:
Hello @pangwa, just pick the one that is right for you so we can move forward. 
Other teams are using other teams library and that is normal. 

https://github.com/AstarNetwork/astar-apps/search?q=TokensAccounts

Reply:
Thanks @pangwa I understand the situation. You can leave the struct as is.